bluecellulab/circuit/circuit_access/bluepy_circuit_access.py

```diff
@@ -0,0 +1,359 @@
+from __future__ import annotations
```

Comment:
Shouldn't there be a copyright header present?

Reply:
Oh thanks let me add.

Reply:
Done in the last push.